developer-docs-site/docs/indexer/legacy/migration.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Migrate to Transaction Stream Service"
+---
+
+This guide contains information on how to migrate to using the Transaction Stream Service if you are currently running a legacy indexer.
+
+The old set up requires running an archival fullnode with additional threads to process the transactions which is difficult and expensive to maintain. Adding more custom logic either requires a bulkier machine, or running several fullnodes that scale linearly.
+
+This new way of indexing requires only a single (or 2 for redundancy) fullnodes. Custom logic live in separate light servers that scale separately from the fullnodes so you don’t need to run multiple fullnodes. You can also directly stream from Labs provided endpoints which means that you no longer need to run fullnodes.
```

Comment:
I wouldn't mention that users have to run fullnodes here, just mention that they have to use the Transaction Stream Service and briefly mention that they can either use our hosted, ratelimited instance or run their own. No need to go too deep since you dive into that choice later on.